conference/src/main/java/com/kyriba/conference/management/domain/Presentation.java

```diff
@@ -1,22 +1,41 @@
 package com.kyriba.conference.management.domain;
 
 
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.ToString;
+import com.kyriba.conference.management.domain.exception.InvalidPresentationTime;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
 import java.time.LocalTime;
 
 
-//@Entity
-@Getter
-@ToString
-@EqualsAndHashCode
+@Entity
+@Data
+@Builder
```

Comment:
Builder is usually used for complex objects with many fields and flexible configuration